docs/Changelog.rst

```diff
@@ -15,6 +15,24 @@ The format is based on `Keep a Changelog`_, and this project adheres to
    :local:
    :depth: 1
 
+.. _v1.1.4:
```

Comment:
I wonder if this change should up the version to 1.2.0 instead as it introduces a larger change than a bug-fix?